File: src/Entity/Societelivraison.php
<?php

namespace App\Entity;

use App\Repository\SocietelivraisonRepository;
use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;

/**
 * @ORM\Entity(repositoryClass=SocietelivraisonRepository::class)
 */
class Societelivraison
{
    /**
     * @ORM\Id
     * @ORM\GeneratedValue
     * @ORM\Column(type="integer")
     */
    private $id;

    /**
     * @ORM\Column(type="string", length=255)
     * @Assert\NotBlank()
     */
    private $nom;

    /**
     * @ORM\Column(type="string", length=255)
     * @Assert\NotBlank()
     */
    private $adresse;

    /**
     * @ORM\Column(type="integer")
     * @Assert\NotBlank()
     * @Assert\Length(
     *      min = 8,
     *      max = 8,
     *      minMessage = "le numero doit etre compose de 8 chiffres",
     *      maxMessage = "le numero doit etre compose de 8 chiffres"
     * )
     */
    private $contact;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getNom(): ?string
    {
        return $this->nom;
    }

    public function setNom(string $nom): self
    {
        $this->nom = $nom;

        return $this;
    }

    public function getAdresse(): ?string
    {
        return $this->adresse;
    }

    public function setAdresse(string $adresse): self
    {
        $this->adresse = $adresse;

        return $this;
    }

    public function getContact(): ?int
    {
        return $this->contact;
    }

    public function setContact(int $contact): self
    {
        $this->contact = $contact;

        return $this;
    }
    /**
     * Transform to string
     *
     * @return string
     */
    public function __toString()
    {
        return (string) $this->getId();
    }
}

File: src/Entities/societelivraison.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package pidevMobile.Entities;

/**
 *
 * @author Abdelalim Mahfoudh
 */
public class societelivraison {
       private int id;
    private String nom;
    private String adresse;
    private int contact;
    
     public societelivraison(){
}
     
     public societelivraison(int id ,String nom, String adresse, int contact  ) {
        this.id=id;
        this.nom=nom;
        this.adresse=adresse;
        this.contact=contact;
        
    }
      public societelivraison(int id ) {
        this.id=id;
       
    }
       public societelivraison(String id ) {
        this.nom=id;
       
    }
      public societelivraison(String nom, String adresse, int contact  ) {
        this.nom=nom;
        this.adresse=adresse;
        this.contact=contact;
        
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNom() {
        return nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    public String getAdresse() {
        return adresse;
    }

    public void setAdresse(String adresse) {
        this.adresse = adresse;
    }

    public int getContact() {
        return contact;
    }

    public void setContact(int contact) {
        this.contact = contact;
    }

    @Override
    public String toString() {
        return "societelivraison{" + "id=" + id + ", nom=" + nom + ", adresse=" + adresse + ", contact=" + contact + '}';
    }



    
    
}
